jetty-servlet/src/main/java/org/eclipse/jetty/servlet/StatisticsServlet.java

```diff
@@ -101,6 +101,12 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
             }
         }
 
+        String statsReset = req.getParameter( "statsReset" );
+        if (statsReset != null && "true".equalsIgnoreCase( statsReset )) {
```

Comment:
These 2 lines can be simplified using `Boolean.valueOf(String)` that also takes care of `null`.
Also, please use the brace and space formatting of the code around, in particular: braces on newline, and no spaces around parenthesis.